src/plugins/kibana/server/routes/api/search/count/register_count.js

```diff
@@ -0,0 +1,34 @@
+import _ from 'lodash';
+import handleESError from '../../../../lib/handle_es_error';
+
+export default function registerCount(server) {
+  const path = '/api/kibana/{id}/_count';
+  const handler = function (req, reply) {
+    const boundCallWithRequest = _.partial(server.plugins.elasticsearch.callWithRequest, req);
+
+    boundCallWithRequest('count', {
+      allowNoIndices: false,
+      index: req.params.id
+    })
+      .then(
+        function (res) {
+          reply({count: res.count});
+        },
+        function (error) {
+          reply(handleESError(error));
+        }
+      );
+  };
+
+  server.route({
+    path,
+    method: 'POST',
```

Comment:
Yes, I even looked that up yesterday, I guess I didn't have enough coffee this morning. Updated.